subtests/docker_cli/systemd/p4321-docker.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=docker-autotest systemd subtest. If you see this file on a live system it means something has failed very badly
+After=docker.service
+
+[Service]
+ExecStart=/usr/bin/docker run -p 4321:4321 --name p4321 p4321
+ExecStop=-/usr/bin/docker stop p4321
+ExecStop=/usr/bin/docker rm -f p4321
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I wonder, does it even ever need to be installed?  IIRC aren't the test requirements just about starting/stopping it?  Maybe I'm remembering wrong.